### ISSUE TITLE
PHP 8.x Deprecated warnings

### ISSUE THREAD

**Issue by ipajen** [GreenerWP/greenerwp-wordpress-plugin#4]
Check the link for PHP warnings on PHP 8.1

[Deprecated warnings on plugintests.com ](https://plugintests.com/plugins/wporg/greenerwp/latest)

+ on PHP 8.2 Creation of dynamic property warnings

Would be great if the plugin could be updated, to remove the warnings.

**Comment by chrneumann** [comment]
Yeah, the included Pimple class has to be updated. Will have a look at it for the next release.
